packages/sdk/src/index.browser.ts

```diff
@@ -30,6 +30,11 @@ export {
 export { createApiClient } from "./apiClient/apiClient";
 export { AuthProvider } from "./apiClient/authProvider";
 export { BearerTokenAuthProvider } from "./apiClient/bearerTokenAuthProvider";
+export {
+  CertificateAuthProvider,
+  createPemCertOption,
+  createPfxCertOption,
+} from "./apiClient/certificateAuthProvider.browser";
```

Comment:
Do we want to have separate certificateAuthProvider file for browser? Or we just leverage the same file?

Reply:
The logic for browser/node is not same, so we need different files